src/app/projects/project-portfolio-wizard/portfolio-review-step/portfolio-review-step.tpl.html

```diff
@@ -20,13 +20,13 @@
   <div class="compile-scheduled text-center text-success" ng-show="portfolioIsCompiling">
     <i class="fa fa-check-circle fa-3x"></i>
     <h3>Compile Scheduled</h3>
-    <p>Your portfolio compilation is scheduled with Doubtfire. This process will take some time, check back soon to see if your portfolio is available.
+    <p>Your portfolio compilation is scheduled with {{ externalName  }}. This process will take some time, check back soon to see if your portfolio is available.
```

Comment:
Double spacing? Might just want to do a search and replace from `{{ externalName   }}` -> `{{externalName}}` (pretty sure angular docs don't space around the double braces).